packages/web/app/src/lib/urql.ts

```diff
@@ -34,6 +35,14 @@ export const urqlClient = createClient({
       updates: {
         Mutation,
       },
+      resolvers: {
+        Target: {
+          appDeployments: relayPagination(),
+        },
+        AppDeployment: {
+          documents: relayPagination(),
+        },
+      },
```

Comment:
This method is way more chilled than the patterns we use in other locations in the code. The drawback is the global configuration...